### ISSUE TITLE
Collection phase failed with an Error 

### ISSUE THREAD

**Issue by bheemvennapureddy** [SteveGilham/altcover#223]
```
Passed:   201, Skipped:     0, Total:   201, Duration: 4 s - CuroAdmin.UnitTests.dll (net8.0)
  ... D:\a\1\s\tests\CuroAdmin.UnitTests\coverage.xml.0.acv (27,011b)
  3,578,198 visits recorded in 00:00:00.0440967 (81,144,349 visits/sec)
  A total of 3,578,198 visits recorded
  Coverage statistics flushing took 35.40 seconds
D:\a\.nuget\cache\Tiger\altcover\8.8.21\build\AltCover.proj(206,5): error :  [D:\a\1\s\tests\CuroAdmin.UnitTests\CuroAdmin.UnitTests.vbproj]
D:\a\.nuget\cache\Tiger\altcover\8.8.21\build\AltCover.proj(206,5): error : ERROR *** Collection phase failed [D:\a\1\s\tests\CuroAdmin.UnitTests\CuroAdmin.UnitTests.vbproj]
D:\a\.nuget\cache\Tiger\altcover\8.8.21\build\AltCover.proj(206,5): error :  [D:\a\1\s\tests\CuroAdmin.UnitTests\CuroAdmin.UnitTests.vbproj]
D:\a\.nuget\cache\Tiger\altcover\8.8.21\build\AltCover.proj(206,5): error :  [D:\a\1\s\tests\CuroAdmin.UnitTests\CuroAdmin.UnitTests.vbproj]
D:\a\.nuget\cache\Tiger\altcover\8.8.21\build\AltCover.proj(206,5): error : Value cannot be null. (Parameter 'value') [D:\a\1\s\tests\CuroAdmin.UnitTests\CuroAdmin.UnitTests.vbproj]
D:\a\.nuget\cache\Tiger\altcover\8.8.21\build\AltCover.proj(206,5): error :  [D:\a\1\s\tests\CuroAdmin.UnitTests\CuroAdmin.UnitTests.vbproj]
D:\a\.nuget\cache\Tiger\altcover\8.8.21\build\AltCover.proj(206,5): error : Details written to D:\a\1\s\tests\CuroAdmin.UnitTests\__Instrumented\AltCover-2024-05-08--17-30-08.log [D:\a\1\s\tests\CuroAdmin.UnitTests\CuroAdmin.UnitTests.vbproj]

##[error
```

Will add the log file

**Comment by bheemvennapureddy** [comment]
[AltCover-2024-05-09--17-56-06.log](https://github.com/SteveGilham/altcover/files/15266141/AltCover-2024-05-09--17-56-06.log)

Logs attached

**Comment by bheemvennapureddy** [comment]
@SteveGilham let me know if you need any more info on this

**Comment by SteveGilham** [comment]
Will do, if I can't figure out the failure path from the stack trace.

**Comment by bheemvennapureddy** [comment]
Steve - if you can share some insights on this that would be helpful. Thanks , Have a great day

**Comment by SteveGilham** [comment]
Having taken a quick look, it's in the part of the process that takes source file names and compiles a list of the containing directories for the `<sources>` element of the document. This would happen if there are (synthetic) methods that have sequence points but don't report a source file.

Simple to fix/work-round in the code.

**Comment by bheemvennapureddy** [comment]
@SteveGilham Can you share some insights on when this fix will be out?

**Comment by SteveGilham** [comment]
Should be resolved in release [v8.8.74](https://www.nuget.org/packages/altcover/8.8.74).

**Comment by bheemvennapureddy** [comment]
Working as expected - thanks for fixing this.
